test/cppsuite/tests/test_template.cpp

```diff
@@ -110,3 +116,5 @@ class test_template : public test_harness::test {
         std::cout << "validate: nothing done." << std::endl;
     }
 };
+
+}
```

Comment:
I don't this we need the namespace until down there? We can limit it to the class `tracking_table_template`?

Reply:
Perhaps not, but I'd prefer to keep it this way. A namespace that stops halfway down the file will be very annoying to debug. This ties in to our avoiding multiple classes per .cpp file discussion.